var/spack/repos/builtin/packages/libxsmm/package.py

```diff
@@ -86,6 +90,9 @@ def install(self, spec, prefix):
         # include symbols by default
         make_args = ['SYM=1']
 
+        if '+header-only' in spec and '@1.6.2:' in spec:
```

Comment:
So for versions before 1.6.2, neither the header or libraries are installed no matter what variants are used. Is this the correct behavior?